### ISSUE TITLE
setup: context cancelled if running long time steps in setup

### ISSUE THREAD

**Issue by gushengyuan** [loadimpact/k6#656]
if I process a large list in the setup(), that will meet "context cancelled" issue, and it's ok if I pick a small sub array from this list using slice(...)

```js
export function setup() {
    let users = JSON.parse(usersList).slice(0, 8);
    var data = [];
    for (var i = 0; i < users.length; i++) {
        let deviceId = "230a361e-25cc-3e40-bf44-c30761386" + format(i, "000.##");
	    var temp = secureLogin(users[i], deviceId, "1");
        data.push(temp);
    }
    return data;
};
```

seams like it's timeout, is there a option to set the expiration time for that?

**Comment by na--** [comment]
The `setup()` and `teardown()` timeouts used to be hardcoded to 10 seconds, but since the recently released version 0.21, there's now an [option to change that](https://github.com/loadimpact/k6/blob/master/release%20notes/v0.21.0.md#configurable-setup-and-teardown-timeouts-602). Try setting this:
```js
export let options = {
  setupTimeout: "60s"
};
```
...or whatever value makes sense in your context. I see that I've forgotten to to add those options in the [documentation](https://docs.k6.io/docs/options, so I'll fix that in a bit...

**Comment by na--** [comment]
Added the `setupTimeout` and `teardownTimeout` to the [options docs](https://docs.k6.io/v1.0/docs/options).
